src/applications/personalization/profile-2/constants.js

```diff
@@ -1,7 +1,17 @@
-export const imagePaths = {
-  Army: '/img/vic-army-symbol.png',
-  'Coast Guard': '/img/vic-cg-emblem.png',
-  'Air Force': '/img/vic-air-force-coat-of-arms.png',
-  Navy: '/img/vic-navy-emblem.png',
-  'Marine Corps': '/img/vic-usmc-emblem.png',
-};
+// The values of these constants map to the possible values that come back from
+// the GET profile/service_history API.
+export const USA_MILITARY_BRANCHES = Object.freeze({
+  army: 'Army',
+  coastGuard: 'Coast Guard',
+  airForce: 'Air Force',
+  navy: 'Navy',
+  marineCorps: 'Marine Corps',
+});
+
+export const SERVICE_BADGE_IMAGE_PATHS = Object.freeze({
+  [USA_MILITARY_BRANCHES.army]: '/img/vic-army-symbol.png',
+  [USA_MILITARY_BRANCHES.coastGuard]: '/img/vic-cg-emblem.png',
+  [USA_MILITARY_BRANCHES.airForce]: '/img/vic-air-force-coat-of-arms.png',
+  [USA_MILITARY_BRANCHES.navy]: '/img/vic-navy-emblem.png',
+  [USA_MILITARY_BRANCHES.marineCorps]: '/img/vic-usmc-emblem.png',
+});
```

Comment:
I don't use these enough, but this could be a Map ->

```js
new Map([
   [USA_MILITARY_BRANCHES.army, '/img/vic-army-symbol.png'],
   [USA_MILITARY_BRANCHES.coastGuard, '/img/vic-cg-emblem.png'],
   // etc
])
```

Reply:
hmm... _googles js maps_

Reply:
Seems to make sense from a semantic point of view, although I won't be using any of Map's features. Am curious if @jbalboni or @rianfowler has any thoughts on the matter.

Reply:
I pretty rarely use Maps, since I typically don't need their features, and seeing that something is a Map doesn't tell you any more about what the code is doing than using an object does (in contrast to Set, which I think does communicate a lot more than a plain object). But that's largely habit at this point, using Map more is probably a good thing.

Reply:
I'll do it. I see no harm. Maybe we'll start a new trend. #eriksfirstmap